tutorials/R/Hyperspectral/Intro-hyperspectral/Work-With-Hyperspectral-Data-In-R/Work-With-Hyperspectral-Data-In-R.Rmd

```diff
@@ -452,7 +462,7 @@ what's called an `image stretch`. We will learn how to stretch our image data,
 later. For now, let's plot the values as the log function on the pixel 
 reflectance values to factor out those larger values. 
 
-```{r plot-log }
+```{r plot-log, fig.cap="Plot of log transformed reflectance values for the previous b9 image." }
```

Comment:
Here, please describe _how_ this image has improved in terms of its information content - and was a log transform enough?